### DISCUSSION TITLE
summarise() point density by class

### DISCUSSION THREAD

**Discussion by summitbri** [r-lidar/lasR#107] [Q&A]
Is this possible with the metrics engine?  Or is there an las area statistic in lasR where I could just do the math using npoints by class

Thanks!

**Comment by Jean-Romain** [comment]
This is something easy to do with `lidR`. The scope of `lasR` is different but this is something that could be added in `summarise()`.

**Reply by summitbri** [reply]
Thanks, I've done it in lidR but this would be handy to have in summarise() someday.
